### DISCUSSION TITLE
c++ client for ByConity, thanks

### DISCUSSION THREAD

**Discussion by 1261385937** [ByConity/ByConity#353] [Q&A] [answered]
Can clickhouse-cpp be used for ByConity as c++ client library ?
Or where is the official c++ client library ? 
Thank you for reply 

@hustnn

**Comment by hustnn** [comment, marked as answer]
Hi, you can download and install official client here https://github.com/ByConity/ByConity/releases/tag/0.1.0-GA.
The Clickhouse client can also be used.
